indico/modules/events/sessions/__init__.py

```diff
@@ -56,3 +62,8 @@ class CoordinatorRole(ManagementRole):
     name = 'coordinate'
     friendly_name = _('Coordination')
     description = _('Grants coordination access to the session.')
+
+
+@signals.menu.items.connect_via('event-management-sidemenu')
+def _extend_event_management_menu(sender, event, **kwargs):
+    return SideMenuItem('sessions', _('Sessions'), url_for('sessions.session_list', event), section='organization')
```

Comment:
```python
def can_manage_sessions(user, event, role=None):
    """Check whether a user can manage any sessions in an event"""
    return any(s.can_manage(user, role) for s in event.sessions.options(joinedload('acl_entries')))
```

(probably something for `util.py`)

Then you can add this check here to avoid showing the menu entry if the user cannot manage any sessions (e.g. someone who has limited management access only):

    if not can_manage_sessions(session.user, event, 'ANY'):
        return